packages/link/src/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./Link";
```

Comment:
thoughts on how we export components from packages? right now, a consumer would import this as:

```javascript
import Link from "@dsco_/link";
```

which is probably fine for most standalone components. however, we'll likely end up having "sets" of components that should live together in a single package, like form components:

```javascript
import { Field, FieldGroup } from "@dsco_/form";

// alternatively, we could do something like:
import Form from "@dsco_/form";

function MyForm() {
  return <Form.FieldGroup>
    <Form.Field />
  </Form.FieldGroup>
}
```

basically, i'm wondering if we have a preference around default vs. named exports. i'd like to be as consistent as possible, and we'll need documentation for all of these, but it's always nice when you can "just guess" when importing because we're following a set of conventions

Reply:
In the code-dot-org repo we use a lot of default exports, but my personal preference is for using only named exports so that you don't need to know remember which way the source is exporting when you're using it, you just always use named exports.

Reply:
that makes sense to me. my other worry with default exports is that it would be annoying/confusing if we later need to export another component from that package -- do you change your default to a named export? do you mix default with named exports? i'd hate to have this situation:

```javascript
import Link, { SomeSpecializedLink } from "@dsco_/link";
```

so it seems cleanest to go with all named exports